### ISSUE TITLE
Keybinding conflicts for sending selected text

### ISSUE THREAD

**Issue by patrickpang** [jeremyramin/terminal-plus#139]
Isn't the keybinding conflicts with the builtin one?
```
'atom-workspace atom-text-editor:not([mini])':
  'ctrl-enter': 'editor:newline-below'
```

**Comment by jeremyramin** [comment]
Yes you are right. I should adjust this to be something else for Windows and Linux machines. Do you have any suggestions that would work best for you?

**Comment by jlroo** [comment]
I am using  ```ctrl-`': 'terminal-plus:insert-selected-text' as``` as I usually just open the terminal for the icon.
